docs(window): added information about the window footer section (#1462)
telerik/blazor-docs: components/window/overview.md

1. Add content to the `WindowContent` child tag.
1. (optional) Add some title inside a `WindowTitle` tag. HTML markup and child components are supported, too.
1. (optional) Add a [`Close` action]({%slug components/window/actions%}) inside a `<WindowActions>` tag.

> 1. (optional) Add content within the `WindowFooter` tag to include custom content in the section rendered at the bottom.

Reviewer: ```suggestion
1. (optional) Add a `WindowFooter` tag to include custom content in the bottom section of the Window.
```

| `Size` | `string` | A predefined Window **width**. Use the string members of the static class `ThemeConstants.Window.Size` - `Small`, `Medium`, and `Large`. They translate to widths of `300px`, `800px` and `1200px`, respectively. If set, the `Width` parameter will take precedence over `Size`. |
| `ThemeColor` | `string` | A predefined color scheme for the Window, especially the titlebar. Use the available members of the static class [`ThemeConstants.Window.ThemeColor`](/blazor-ui/api/Telerik.Blazor.ThemeConstants.Window.ThemeColor). |
| `Visible` | `bool` | Defines if the Window is rendered and visible on the page. |

> | `FooterLayoutAlign` | `WindowFooterLayoutAlign` enum <br /> (`Stretch`) | Controls the alignment of the items in the `WindowFooter`. Takes a member of the `WindowFooterLayoutAlign` enum. The possible options are `Stretch`, `Start`, `End`, and `Center`. |

Reviewer: ```suggestion
| `FooterLayoutAlign` | `WindowFooterLayoutAlign` enum <br /> (`Stretch`) | Controls the alignment of the HTML elements in the `WindowFooter`. Takes a member of the `WindowFooterLayoutAlign` enum. The possible options are `Stretch`, `Start`, `End`, and `Center`. |
```
